src/osiris_log.erl

```diff
@@ -1811,7 +1881,8 @@ timestamp_idx_scan(Fd, Ts) ->
          <<ChunkId:64/unsigned,
            Timestamp:64/signed,
            Epoch:64/unsigned,
-           FilePos:32/unsigned>>} ->
+           FilePos:32/unsigned,
+           ?CHNK_USER:8/unsigned>>} ->
```

Comment:
this will crash if another type of entry is in the index. I'd suggest to add a clause to filter out the non-user entries.

Reply:
Good catch, we don't filter by this chunk type anymore as suggested by Karl below.